AlphaWallet/Settings/ViewControllers/SettingsViewController.swift

```diff
@@ -107,6 +107,30 @@ class SettingsViewController: UIViewController {
         }
     }
 
+    private func configureChangeWalletCellWithResolvedESN(_ row: SettingsWalletRow, cell: SettingTableViewCell) {
+        let viewModel: SettingTableViewCellViewModel = .init(
+            titleText: row.title,
+            subTitleText: account.address.eip55String,
+            icon: row.icon
+        )
+
+        let serverToResolveEns = RPCServer.main
+        let address = account.address
+
+        ENSReverseLookupCoordinator(server: serverToResolveEns).getENSNameFromResolver(forAddress: address) { result in
+            guard let ensName = result.value else { return }
+
+            let viewModel: SettingTableViewCellViewModel = .init(
+                titleText: row.title,
+                subTitleText: ensName,
```

Comment:
The `subTitleText` should be include the Ethereum address. Refer to `AccountViewModel.addresses` to see how it should be displayed.